es6-mythical-creatures/test/fairy-test.js

```diff
@@ -0,0 +1,134 @@
+var assert = require('chai').assert;
+var Fairy = require('../exercises/fairy');
```

Comment:
Looks like this file is still es5

Reply:
Good catch, I'll get that fixed.  Thanks David! 👍